tools/transform.cpp

```diff
@@ -432,7 +485,23 @@ static void calculateAndInitConstants(Transform &t) {
 
   // TODO
   bits_for_offset = 64;
-  bits_size_t = 64;
+
+  // size of byte
+  if (num_nonlocals != 1 || t.src.getType().isPtrType())
+    // Be conservative, run when there are unescaped local blocks only.
```

Comment:
what's the problem with functions returning pointers or with global variables or with pointer inputs?
If all memory accesses are aligned, it would be UB if the program receives an unaligned ptr that is dereferenced. So I don't see why this restriction is needed?

Reply:
Yes, you're right. This part had to be deleted.